src/lib/isBefore.js

```diff
@@ -1,9 +1,12 @@
-import assertString from './util/assertString';
 import toDate from './toDate';
 
-export default function isBefore(str, date = String(new Date())) {
-  assertString(str);
-  const comparison = toDate(date);
+export default function isBefore(str, options) {
+  // accessing 'arguments' for backwards compatibility: isBefore(str [, comparisonDate])
+  // eslint-disable-next-line prefer-rest-params
+  const comparisonDate = (typeof options === 'object' ? options.comparisonDate : arguments[1]) || Date().toString();
```

Comment:
```suggestion
  // For backwards compatibility:
  // isBefore(str [, date]), i.e. `options` could be used as argument for the legacy `date`
  const comparisonDate = options?.comparisonDate || options || Date().toString();
```

Would this work as well? Copied from `isAfter`

Reply:
Nope, that would be a bug :(.

If `comparisonDate` is undefined, the ternary operator will therefore evaluate to `options`, which would be `{ comparisonDate: undefined }`. The latter will break when using inside `toDate`, because it is not a string. 

Well spotted anyway. This way I realised that I wanted to add tests to cover this scenario.

Reply:
That's the case in `isAfter` as well right? So we can fix it for both validators in this PR I think

Reply:
Yes, that would be the case. I will add a fix.

Reply:
I've fixed `isAfter` and added tests. FYI, the last test would fail with the old way in `isAfter`.